data/application_store/new_application.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Performance Test Run",
+    "questions": {
```

Comment:
questions is a list of question dictionaries : so should be:
'questions' : [
{
       "question": "Performance Test Run",
       "fields": {},
       "category": "Performance Test Run",
       "index": 0,
       "status": "Performance Test Run"
     }
]

Reply:
This comment is the reason it is getting an internal server error. I have changed it to that and now it works. Might want to have a look at the code as to why it gives a 500 when you pass a questions object instead of an array. Good spot

Reply:
Agreed, ill make a note now to investigate that. Good spot Damien 👀